src/components/datagrid/data_grid_body.tsx

```diff
@@ -554,6 +558,13 @@ export const EuiDataGridBody: FunctionComponent<EuiDataGridBodyProps> = (
   );
 
   const [minRowHeight, setRowHeight] = useState(INITIAL_ROW_HEIGHT);
+
+  useEffect(() => {
+    if (gridRef.current) {
+      rowHeightUtils.setGrid(gridRef.current);
+    }
+  });
```

Comment:
Should pass an empty dependency array to avoid firing this every render

```suggestion
  }, []);
```

Reply:
We don't sure that gridRef was set before useEffect which have empty dependency will call (aka didMount).

Reply:
Maybe adding `gridRef.current` then? 
```suggestion
  }, [gridRef.current]);
```

Reply:
Now I'm using callback ref for set grid instance in utils instead of useEffect.